docs/usage.md

```diff
@@ -3891,10 +3967,621 @@ nextflow clean -f -k
+  the first few nucleotide positions?) ? If not, possibly your mapped
+  reads are deriving from modern contamination.
+
+QualiMap:
+
+- Do you see a peak of coverage (X) at a good level, e.g. >= 3x, indicating
+  sufficient coverage for reliable SNP calls?
+
+MultiVCFAnalyzer:
+
+- Do I have a good number of called SNPs that suggest the samples have genomes
+  with sufficient nucleotide diversity to inform phylogenetic analysis?
+- Do you have a large number of discarded SNP calls?
+- Are the % Hets very high?
+
+> Detailed documentation and descriptions for all MultiQC modules can be seen in
+> the the 'Documentation' folder of the results directory or here in the [output
+> documentation](output.md)
+
+If you're happy everything looks good in terms of sequencing, we then look at
+specific directories to find any files you might want to use for downstream
+processing.
+
+Note that when you get back to writing up your publication, all the versions of
+the tools can be found under the 'nf-core/eager Software Versions' section of
+the MultiQC report. Note that all tools in the container are listed, so you may
+have to remove some of them that you didn't actually use in the set up.
+
+For example, in this example, we have used: Nextflow, nf-core/eager, FastQC,
+AdapterRemoval, fastP, BWA, Samtools, endorS.py, Picard Markduplicates,
+Bedtools, Qualimap, PreSeq, DamageProfiler, MultiVCFAnalyzer and MultiQC.
+
+Citations to all used tools can be seen
+[here](https://nf-co.re/eager#tool-references)
+
+##### Tutorial Pathogen Genomics - Files for Downstream Analysis
+
+You will find the most relevant output files in your `results/` directory. Each
+directory generally corresponds to a specific step or tool of the pipeline. Most
+importantly you should look in `deduplication` for your de-duplicated BAM files
+(e.g. for viewing in IGV), bedtools for depth (X) and breadth (%) coverages of
+annotations of your reference (e.g. genes), `multivcfanalyzer` for final SNP
+tables etc.
+
+#### Tutorial Pathogen Genomics - Clean up
+
+Finally, I would recommend cleaning up your `work/` directory of any
+intermediate files (if your `-profile` does not already do so). You can do this
+by going to above your `results/` and `work/` directory, e.g.
+
+```bash
+cd /<path>/<to>/projectX_preprocessing20200727
+```
+
+and running
+
+```bash
+nextflow clean -f -k
+```
+
+#### Tutorial Pathogen Genomics - Summary
+
+In this this tutorial we have described an example on how to set up an
+nf-core/eager run to process microbial aDNA for a relatively standard pathogen
+genomics study for phylogenetics and basic functional screening. This includes
+preform some simple quality control checks, mapping, genotyping, and SNP table
+generation for downstream analysis of the data. Additionally, we described what
+to look for in the run summary report generated by MultiQC and where to find
+output files that can be used for downstream analysis.
```

Comment:
Want to add a warning that this showcases the functionality of the pipeline, but the shown parameters/settings are not the ones which are normally used, like you would not want to have het calls in your SNP alignment for phylogenetic inference.